docusaurus.config.js

```diff
@@ -2,19 +2,17 @@ const path = require('path')
 const fs = require('fs')
 
 // Change this value to update what the un-versioned docs url should be
-const unreleasedTauriVersion = 'v1'
-var lastestReleasedVersion
+const unreleasedTauriVersion = 'v2'
+let lastReleasedVersion
 
 // Checks if Docusaurus has been versioned before and sets versions accordingly
 try {
-  lastestReleasedVersion = JSON.parse(
-    fs.readFileSync('versions.json', 'utf-8')
-  )[0]
+  lastReleasedVersion = JSON.parse(fs.readFileSync('versions.json', 'utf-8'))[0]
```

Comment:
Now that you'll actually have this `versions.json` file generated and committed, you can remove line 6 above and remove the conditionality on 309-312 below. They were just there since you don't have the versions.json file until you've first made a version cut